### PR TITLE
add datetime intl module

### PR DESCRIPTION
**Pull request by embbnux**
add datetime intl module

### CODE REVIEW

File: src/modules/DateTimeIntl/getStorageReducer.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { combineReducers } from 'redux';
+
+export function getSettingsReducer(actionTypes) {
+  return (state = {}, { type, provider, providerSettings }) => {
+    switch (type) {
+
+      case actionTypes.fetchSuccess:
+        state[provider.providerName] = providerSettings;
+        return state;
```
Comment (LingForCC) [inline src/modules/DateTimeIntl/getStorageReducer.js]: Should return a new object other than the old one.

**Comment by embbnux** [comment]
rerun

**Comment by embbnux** [comment]
rerun

**Comment by embbnux** [comment]
requested changes have fixed, wait for review

**Pull request merged**